cmd/crictl/config.go

```diff
@@ -66,37 +71,46 @@ var configCommand = &cli.Command{
 				logrus.Fatalf("No section named %s", get)
 			}
 			return nil
-		}
-		key := context.Args().First()
-		if key == "" {
-			return cli.ShowSubcommandHelp(context)
-		}
-		value := context.Args().Get(1)
-		switch key {
-		case "runtime-endpoint":
-			config.RuntimeEndpoint = value
-		case "image-endpoint":
-			config.ImageEndpoint = value
-		case "timeout":
-			n, err := strconv.Atoi(value)
-			if err != nil {
-				logrus.Fatal(err)
+		} else if context.IsSet("set") {
+			settings := context.StringSlice("set")
+			for _, setting := range settings {
+				options := strings.Split(setting, ",")
+				for _, option := range options {
+					pair := strings.Split(option, "=")
+					if len(pair) != 2 {
+						return fmt.Errorf("incorrectly specified option: %v", setting)
+					}
+					key := pair[0]
+					value := pair[1]
+					switch key {
+					case "runtime-endpoint":
```

Comment:
can we make these keys a list and dump them into the usage for config?

Reply:
Added the options to the description.